### PR TITLE
about page code is converted to typescript support code

### PR DESCRIPTION
**Pull request by Abdullatif-Erfan**

### CODE REVIEW

File: src/pages/about.tsx [hunk 1]
```diff
@@ -1,4 +1,7 @@
- function about(){
-    return <h1>about page</h1>
+import { NextPage } from "next";
+const about: NextPage<{}> = () => {
```
Comment (AjmalNasiri) [inline src/pages/about.tsx:3]: in thise part pleae remove the complex part and add simple form of the code

**Review by AjmalNasiri — CHANGES_REQUESTED** [review]
information for about page changes

File: src/pages/about.tsx (hunk 1 above)
Comment (Abdullatif-Erfan) [reply src/pages/about.tsx:3]: I have add a comment on line 2 of about page for clirification

Comment (AjmalNasiri) [reply src/pages/about.tsx:3]: ik thank you it is approved and merge it

**Review by AjmalNasiri — APPROVED** [review]
ok done complete

**Comment by AjmalNasiri** [comment]
mege abdul latif changes and finall version approved

**Pull request merged**